projects/sbb-esta/angular-business/status/src/status/status.component.html

```diff
@@ -0,0 +1,20 @@
+<ng-template #content><ng-content></ng-content></ng-template>
```

Comment:
The design specification (currently) specifies the states valid, warning, invalid (and consumer specific variants). I would like to provide these by default without having the consumer provide the icons themselves. The consumer should only provide the icons for consumer specific variants.